lib/widgets/pin_code_widget.dart

```diff
@@ -27,9 +34,24 @@ class PinCodeWidgetState extends State<PinCodeWidget> {
     super.initState();
     _enteredPinCode = "";
     _errorMessage = "";
+    if (widget.onFingerprintEntered != null) {
+      widget.userProfileBloc.userStream.firstWhere((u) => u.securityModel.enrolledBiometrics != "").then((_) {
+        _validateBiometrics();
```

Comment:
We also need to put some delay here since the screen is shown with fade transition and the touch id is shown immediately halting the UI causes the screen to be  half transparent.

Reply:
I pushed this change.

Reply:
Looks good

Reply:
[We listen to first item available to avoid unwanted behaviour on Security & Backup page](https://github.com/breez/breezmobile/pull/210/commits/5b360eeda4c134c5b432eaa2aa0bd11d6b8c8014)